libs/preprocessor/README.rst

```diff
@@ -0,0 +1,20 @@
+..
+   Copyright (c) 2019 The STE||AR-Group
+
+   Distributed under the Boost Software License, Version 1.0. (See accompanying
+   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+===
+preprocessor
+===
+
+This library contains useful preprocessor macros:
+
+* ``HPX_PP_CAT``: Concatenate two tokens
+* ``HPX_PP_EXPANDS``: Expands a preprocessor token
+* ``HPX_PP_NARGS``: Determines the number of arguments passed to a variadic macro
+* ``HPX_PP_STRINGIZE``: Turns a token into a string
+* ``HPX_PP_STRIP_PARENS``: Strips parenthesis from a token
+
+Extensive documentation can be found `here
+<https://stellar-group.github.io/hpx/docs/sphinx/latest/html/libs/pp/docs/index.html>`__.
```

Comment:
This is not true ;-)

Reply:
Damn it, I changed it to say just `Documentation can be found...`. Even that's a bit of an overstatement. I promise that's the last thing I'll change and then I think we can merge this.